Magento_Cms/web/js/custom-country-form.js

```diff
@@ -0,0 +1,66 @@
+define(['jquery', 'mage/url', 'jquery-ui-modules/widget', 'jquery/validate'], (
+  $,
+  urlBuilder
+) => {
+  $.widget('vaimo.countryFormWidget', {
+    options: {},
+
+    _create() {
+      this.submitForm();
+    },
+
+    submitForm() {
+      const self = this;
+
+      $('#search-country-form').on('submit', function (e) {
```

Comment:
it'd be better to create more methods with a single responsibility, don't write everything in one method with a lot of code and different logic. it's hard to read, maintain and mixin